cloudgarden-pwa/src/AppWithRouterAccess.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Route, useHistory } from "react-router-dom";
+import { Security, SecureRoute, LoginCallback } from "@okta/okta-react";
+import Login from "./components/main/LoginForm";
+import App from "./App";
+
+const AppWithRouterAccess = () => {
+  const history = useHistory();
+  const onAuthRequired = () => {
+    history.push("/login");
+  };
+
+  return (
+    <Security
+      issuer={`${process.env.REACT_APP_OKTA_ORG_URL}/oauth2/default`}
+      clientId={process.env.REACT_APP_OKTA_CLIENT_ID}
+      redirectUri={window.location.origin + "/implicit/callback"}
+      onAuthRequired={onAuthRequired}
+      pkce={true}
+    >
+      <SecureRoute exact path="/" component={App} />
```

Comment:
My thought process here is to make the app root secure, so it redirects to the login page if the user isn't already authenticated